pkg/controllers/resources/persistentvolumeclaims/syncer.go

```diff
@@ -43,10 +56,18 @@ func New(ctx *synccontext.RegisterContext) (syncertypes.Object, error) {
 		return nil, err
 	}
 
+	physicalClusterClient, err := kubernetes.NewForConfig(ctx.PhysicalManager.GetConfig())
```

Comment:
do not construct new client here, use `ctx.PhysicalManager.GetClient()`

Reply:
Done!